packages/web-components/src/components/masthead/__stories__/masthead.stories.ts

```diff
@@ -35,6 +35,21 @@ const platformData = {
   url: 'https://www.ibm.com/cloud',
 };
 
+const platformObject = {
+  'en-US': {
+    url: 'https://www.cloud.com/us-en',
+    name: 'Cloud',
+  },
+  'fr-FR': {
+    url: 'https://www.cloud.com/fr-fr/sample',
+    name: 'Nuage',
+  },
+  'es-MX': {
+    url: 'https://www.cloud.com/ibm/sample/es-mx',
+    name: 'Nube',
```

Comment:
@jeffchew do you know if other platform names should be translated? I know for `Cloud` we don't want it translated.

Reply:
Name doesn't apply anymore 👍